package.json

```diff
@@ -12,6 +12,7 @@
     "@types/jest": "^26.0.14",
     "@types/react": "^16.9.51",
     "@types/react-dom": "^16.9.8",
+    "@types/react-select": "^3.0.26",
```

Comment:
We will move this types to `@hazelcast/ui` later in scope of https://github.com/hazelcast/frontend-shared/issues/59